stream/src/main/scala/org/apache/pekko/stream/scaladsl/Flow.scala

```diff
@@ -2804,6 +2804,25 @@ trait FlowOps[+Out, +Mat] {
   def flattenMerge[T, M](breadth: Int)(implicit ev: Out <:< Graph[SourceShape[T], M]): Repr[T] =
     flatMapMerge(breadth, ev)
 
+  /**
+   * Transforms each input element into a `Source` of output elements that is
+   * then flattened into the output stream until a new input element is received
+   * at which point the current (now previous) substream is cancelled (which is why
+   * this operator is sometimes also called "flatMapLatest").
+   *
+   * '''Emits when''' the current substream has an element available
+   *
+   * '''Backpressures when''' never
```

Comment:
this `never` is really great!

Reply:
On second thought: this might be formally correct as it never backpressures on upstream (and backpressure on upstream is what is to be adressed here if I'm not mistaken), but maybe it would be more helpful to mention the backpressure on substream, e.g.:

```suggestion
   * '''Backpressures when''' never on upstream (backpressures on current substream when downstream backpressures)
```
wdyt?